### PR TITLE
Feat: Chapter Web Views Added!

### PR DESCRIPTION
**Pull request by harshtandiya**
Chapter Web Views are finally shipped! 🥳
Designs are as per decided on Figma. Find screenshots below.

### Title, Location & Social Links
image
- Title will be Press2P font for now.
- The chapter image is dynamic, changes with respect to city chapter/student club
- Location is a mandatory field in the chapter doctype
- Social links support 6 platforms, including: Email, X, Mastodon, Facebook, Instagram and LinkedIn

### About
image

### Upcoming Events
image
- Must Attend is toggle-able in the event doctype as per requirement.
- This will have: Date of event, Event Name & Venue

### Past Events
image

### Team
image
- Team images will be grayscale by default in events and chapters page
- If no display image, Initials are used to make an avatar

## Chapters
image

---

## Web Templates Added
- [x] FOSS Club Lead Role
- [x] FOSS Club - Benefits For The Lead 
- [x] FOSS Club - Support from FOSS United
- [x] FOSS Club - Criteria for Lead
- [x] FOSS Club - Criteria and Responsibilities
- [x] FOSS Club - CTA

### CODE REVIEW

File: fossunited/fossunited/web_template/foss_club___criteria_and_responsibitities/foss_club___criteria_and_responsibitities.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+          <li><b>Partnership Encouragement:</b> Collaborations with groups and companies for additional resources like speakers and venues are welcomed.</li> <br/>
+          <li><b>FOSS Promotion:</b> Avoid promoting non-FOSS entities through the club platform.</li> <br/>
+          <li><b>Brand Alignment:</b> Adhere to the brand guidelines as closely as possible.</li> <br/>
+          <li><b>Active Engagement</b>: Maintain regular activity and consistent interaction with the community.</li><br/>
+          <li><b>Open Communication:</b> Be willing and available to communicate with the FOSS United Team as needed.</li>
+        </ul>
+        
+      </div>
+      
+      <div class="col-md-5 offset-md-2">
```
Comment (NagariaHussain) [inline fossunited/fossunited/web_template/foss_club___criteria_and_responsibitities/foss_club___criteria_and_responsibitities.html]: Check the file name 😆

File: fossunited/fossunited/web_template/foss_club___criteria_and_responsibitities/foss_club___criteria_and_responsibitities.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+ "__unsaved": 1,
+ "creation": "2023-08-11 23:56:43.239940",
+ "docstatus": 0,
+ "doctype": "Web Template",
+ "fields": [],
+ "idx": 0,
+ "modified": "2023-08-11 23:56:43.239940",
+ "modified_by": "Administrator",
+ "module": "FOSSUnited",
+ "name": "FOSS Club - Criteria and Responsibitities",
```
Comment (NagariaHussain) [inline fossunited/fossunited/web_template/foss_club___criteria_and_responsibitities/foss_club___criteria_and_responsibitities.json]: Typo

**Review by NagariaHussain — CHANGES_REQUESTED** [review]

File: fossunited/fossunited/web_template/foss_club___criteria_and_responsibitities/foss_club___criteria_and_responsibitities.html (hunk 1 above)
Comment (harshtandiya) [reply fossunited/fossunited/web_template/foss_club___criteria_and_responsibitities/foss_club___criteria_and_responsibitities.html]: 🤣🤣🤣🤣 lmfaoooo my bad

File: fossunited/fossunited/web_template/foss_club___criteria_and_responsibitities/foss_club___criteria_and_responsibitities.json (hunk 2 above)
Comment (harshtandiya) [reply fossunited/fossunited/web_template/foss_club___criteria_and_responsibitities/foss_club___criteria_and_responsibitities.json]: @NagariaHussain Resolved

**Pull request merged**